dev/reference/tools/data-migration/cluster-operations.md

```diff
@@ -489,23 +489,52 @@ Assuming that the `172.16.10.72` machine needs to be maintained or this machine
+## Switch DM-worker connection between upstream MySQL instances
 
-This section describes how to switch between master and slave instances using dmctl in two conditions.
+When the upstream MySQL instance that DM-worker connects to needs a downtime maintenance or when the instance crashes unexpectedly, you need to switch the DM-worker connection to another MySQL instance within the same replication group.
 
-### Upstream master-slave switch behind the virtual IP
+> **Note:**
+>
+> - You can only switch the DM-worker connection to a instance within the same master-slave replication cluster.
+> - The MySQL instance to be newly connected to must have the binlog required by DM-worker.
+> - DM-worker must operate in the GTID sets mode. That is to say that you must specify `enable_gtid=true` when you deploy DM using DM-Ansible.
+> - The connection switch only supports the following two scenarios. Strictly follow the procedures for each scenario. Otherwise, you might have to re-deploy the DM cluster according to the newly connected MySQL instance and perform the data replication task all over again.
+
+For more details on GTID set, refer to [MySQL documentation](https://dev.mysql.com/doc/refman/5.7/en/replication-gtids-concepts.html#replication-gtids-concepts-gtid-sets).
 
-1. Use `query-status` to make sure that the relay unit has caught up with the binlog status of the master instance before the switch (`relayCatchUpMaster`).
-2. Use `pause-relay` to pause relay.
-3. Use `pause-task` to pause all running tasks.
-4. The upstream master and slave instances behind the virtual IP execute the switch operation.
-5. Use `switch-relay-master` to tell relay to execute the master-slave switch.
-6. Use `resume-relay` to make relay resume to read binlog from the new master instance.
-7. Use `resume-task` to resume the previous replication task.
+### Switch DM-worker connection behind virtual IP
 
-### Master-slave switch after changing IP
+When DM-worker connects the upstream MySQL instance behind virtual IP (VIP) and if you switch the VIP connection to another MySQL instance, you switch the MySQL instance actually connected to DM-worker with the upstream connection address unchanged.
 
-1. Use `query-status` to make sure that the relay unit has caught up with the binlog status of the master instance before the switch (`relayCatchUpMaster`).
-2. Use `stop-task` to stop all running tasks.
-3. Modify the DM-worker configuration, and use DM-Ansible to perform a rolling update on DM-worker.
-4. Use `start-task` to restart the replication task.
+> **Note:**
+>
+> Make necessary changes on DM in this scenario. Otherwise, when you switch the VIP connection to another MySQL instance, DM might connect to the new and old MySQL instances at the same time in different connections. In this situation, the binlog replicated to DM is not consistent with other upstream status that DM receives, causing unpredictable anomalies and even data damage.
```

Comment:
```suggestion
> Make necessary changes to DM in this scenario. Otherwise, when you switch the VIP connection to another MySQL instance, DM might connect to the new and old MySQL instances at the same time in different connections. In this situation, the binlog replicated to DM is not consistent with other upstream status that DM receives, causing unpredictable anomalies and even data damage.
```